config/database.yml

```diff
@@ -18,8 +18,9 @@ development:
   adapter: postgresql
   encoding: unicode
   database: CloneBook_development
+  host: localhost
   pool: 5
-  username: paul
+  username: admin
```

Comment:
It is better to not include username and password, it should be autodetected on other devs machine.

Reply:
We haven't been able to get the app to run correctly without supplying the username and password in this file. Any tips?